services/patient.services.js

```diff
@@ -35,6 +35,42 @@ const createPatient = async ({
   }
 };
 
+const editPatient = async ({
```

Comment:
if the data is already validated (it should be) you can pass two params here:
- slug
- data as an object (image, first name, last name ...)

Reply:
Why you resolved this?